lib/xeroizer/partner_application.rb

```diff
@@ -42,5 +42,14 @@ def initialize(consumer_key, consumer_secret, path_to_private_key, path_to_ssl_c
         end
       end
     
+    private
+
+      def read_certificate(cert)
+        if cert[-4..-1] == '.pem'
```

Comment:
It I remember correctly this was to deal with people using the wrong key
file out of cert, pfx, etc.

On Monday, March 23, 2015, CloCkWeRX <notifications@github.com> wrote:

> In lib/xeroizer/partner_application.rb
> <https://github.com/waynerobinson/xeroizer/pull/175#discussion_r26912337>:
>
> > @@ -42,5 +42,14 @@ def initialize(consumer_key, consumer_secret, path_to_private_key, path_to_ssl_c
> >          end
> >        end
> >
> > +    private
> > +
> > +      def read_certificate(cert)
> > +        if cert[-4..-1] == '.pem'
>
> Why not file.exists? instead. If you have a valid private key but have
> renamed it, this will fail.
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/waynerobinson/xeroizer/pull/175/files#r26912337>.
>